src/extension.ts

```diff
@@ -14,14 +14,17 @@ export async function activate(context: vscode.ExtensionContext) {
   context.subscriptions.push(
     vscode.commands.registerCommand(
       "command-server.runCommand",
-      (focusedElementType_?: FocusedElementType) => {
+      (focusedElementType_: FocusedElementType) => {
         focusedElementType = focusedElementType_;
-        return commandRunner.runCommand();
+        const ret = commandRunner.runCommand();
+        focusedElementType = undefined;
+
+        return ret;
       }
     ),
     vscode.commands.registerCommand(
       "command-server.getFocusedElementType",
-      () => focusedElementType ?? null
```

Comment:
@AndreasArvidsson why did we return `null` here instead of `undefined`?

Reply:
Good question.